tutorials/ES/02_ml-demos-con-gradio.ipynb

```diff
@@ -0,0 +1,556 @@
+        "\n",
+        "\n",
+        "**Autor**: [Abubakar Abid](https://twitter.com/abidlabs) (siéntase libre de enviarme un ping con cualquier pregunta sobre este tutorial) \n",
+        "\n",
+        "¡Todos estos pasos se pueden hacer gratis! Todo lo que necesita es un navegador de Internet y un lugar donde pueda escribir Python 👩‍💻"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "PR9faV2NWTrG"
+      },
+      "source": [
+        "## ¿Por qué demos?\n",
+        "\n",
+        "Los **demos** de modelos de Machine Learning son una parte cada vez más importante de los _cursos_ y _conferencias_ de Machine Learning. Los demos permiten:\n",
+        "* que las personas que desarrollan modelos puedan **presentar** fácilmente su trabajo a una amplia audiencia\n",
+        "* aumentar la **reproducibilidad** de la investigación en Machine Learning\n",
+        "* que divers@s usuari@s puedan **identificar y depurar** más fácilmente los puntos de falla de los modelos\n",
+        "\n",
+        "\n",
+        "Como un ejemplo rápido de lo que nos gustaría construir, eche un vistazo a [Keras org en Hugging Face](https://huggingface.co/keras-io), que incluye una tarjeta de descripción y una colección de Modelos y Spaces construidos por la comunidad de Keras. Cualquier Space se puede abrir en su navegador y puede usar el modelo inmediatamente como se muestra aquí:\n",
+        "\n",
+        "![](https://i.ibb.co/7y6DGjB/ezgif-5-cc52b7e590.gif)\n",
+        "\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "g0KzbU4lQtv3"
+      },
+      "source": [
+        "## 1. Crear demos rápidos de ML en Python usando la biblioteca Gradio"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rlSs72oUQ1VW"
+      },
+      "source": [
+        "`gradio` es una biblioteca de Python que le permite crear demos web simplemente especificando la lista de componentes de entrada y salida que espera su modelo de Machine Learning.\n",
+        "\n",
+        "¿Qué quiero decir con componentes de entrada y salida? Gradio viene con un montón de componentes predefinidos para diferentes tipos de modelos de aprendizaje automático. Aquí hay unos ejemplos:\n",
+        "\n",
+        "* Para un **clasificador de imágenes**, el tipo de entrada esperado es una `Imagen` y el tipo de salida es una `Etiqueta`. \n",
+        "* Para un **modelo de reconocimiento de voz**, el componente de entrada esperado es un `Micrófono` (que permite a los usuarios grabar desde el navegador) o `Audio` (que permite a los usuarios arrastrar y soltar archivos de audio), mientras que el tipo de salida es `Texto`.\n",
+        "* Para un **modelo de respuesta a preguntas**, esperamos **2 entradas**: [`Texto`, `Texto`], un cuadro de texto para el párrafo y otro para la pregunta, y el tipo de salida es un solo `Texto` correspondiente a la respuesta. \n",
```

Comment:
The use of ` means you're referring to code, so it does not make much sense to have `Imagen`. I.e. you can keep original function/input type name